packages/manager/modules/vps/src/dashboard/vps-dashboard.controller.js

```diff
@@ -493,6 +494,10 @@ export default class {
               !this.isVpsNewRange &&
               !this.isMigrationInProgress,
           },
+          migrate: {
+            state: 'vps.detail.dashboard.migrate',
+            stateParams: { serviceName: this.serviceName },
```

Comment:
Shouldn't we use `coreURLBuilder` and provide an `href` instead ?

Reply:
This is the old migration code, has been removed